packages/orchestrator-ui-components/src/rtk/store.ts

```diff
@@ -16,18 +20,26 @@ export type RootState = {
     >;
     toastMessages: ReturnType<typeof toastMessagesReducer>;
     orchestratorConfig: OrchestratorConfig;
+    customBaseQueries?: CustomBaseQuery[];
 };
 
 export const getOrchestratorStore = (
     orchestratorConfig: OrchestratorConfig,
+    customBaseQueries?: CustomBaseQuery[],
```

Comment:
I think I prefer for this to have a default value of []. so below on line 40 you can just always add it but having it empty by default